polyfill.js

```diff
@@ -14,8 +14,10 @@ if (typeof Array.prototype.flatMap !== 'function') {
     // https://tc39.github.io/proposal-flatMap/#sec-Array.prototype.flatMap
     value: function flatMap(callback, thisArg = undefined) {
       const O = ES.ToObject(this);
+      // Could ES.IsCallable check callback, but it's better to let it throw a runtime error
```

Comment:
Polyfills can just make up their own message; a correct observable order of events is more important.